client/my-sites/upgrades/checkout-thank-you/header.jsx

```diff
@@ -68,6 +69,19 @@ const CheckoutThankYouHeader = React.createClass( {
 			} );
 		}
 
+		if ( isGuidedTransfer( this.props.primaryPurchase ) ) {
+			if ( typeof this.props.primaryPurchase.meta === 'string' ) {
+				return this.translate( 'The guided transfer for {{strong}}%(siteName)s{{/strong}} will begin very soon. We will be in touch with you via email.', {
```

Comment:
Consider putting this translation on multiple lines so its more readable without scrolling left-to-right.